akka-actor-typed/src/main/scala/akka/actor/typed/receptionist/Receptionist.scala

```diff
@@ -51,7 +52,7 @@ object ServiceKey {
   /**
    * Scala API: Creates a service key. The given ID should uniquely define a service with a given protocol.
    */
-  def apply[T](id: String)(implicit tTag: ClassTag[T]): ServiceKey[T] =
+  def apply[T: ClassTag](id: String): ServiceKey[T] =
```

Comment:
call me boring but I'd keep the explicit style to be honest... Easier to understand when people dive into the codebase

Reply:
Sure, I did like this because it was a bit duplication with where the logic was also using `implicit[ClassTag[T]]` below, but I can change that to just use `tTag` instead.